docs/bakeries.md

```diff
@@ -8,9 +8,9 @@ Cloud based execution is provided by Bakeries.
 
 There are currently two types of Bakeries implemented:
 
-- <https://github.com/pangeo-forge/pangeo-forge-aws-bakery> -
+- <https://github.com/pangeo-forge-recipes/pangeo-forge-recipes-aws-bakery> -
```

Comment:
So here it looks like you are also changing the name of the github org (`github.com/pangeo-forge`)  and another associated repo (`pangeo-forge-aws-bakery`). We _only_ want to rename this repo, not the org, nor the other project. So I would revert this.